hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -712,8 +736,21 @@ class HassioAddonInfo extends LitElement {
   private _showMoreInfo(ev): void {
     const id = ev.currentTarget.id;
     showHassioMarkdownDialog(this, {
-      title: PERMIS_DESC[id].title,
-      content: PERMIS_DESC[id].description,
+      title: this.supervisor.localize(`addon.dashboard.capability.${id}.title`),
+      content:
+        id === "stage"
+          ? this.supervisor.localize(
+              `addon.dashboard.capability.${id}.description`,
+              "icon_stable",
+              `<ha-svg-icon path="${STAGE_ICON.stable}"></ha-svg-icon>`,
```

Comment:
```suggestion
              html`<ha-svg-icon .path=${STAGE_ICON.stable}></ha-svg-icon>`,
```

Reply:
That will not work.

Reply:
showHassioMarkdownDialog does not support it, will handle that later